ci/pipelines/e2e-helpers.yml

```diff
@@ -293,35 +303,44 @@ jobs:
       put: slack-notification
       attempts: 10
       params:
-        channel: '#govuk-pay-starling'
+        channel: "#govuk-pay-starling"
         silent: true
-        text: ':red-circle: Failed to build and push pay-endtoend candidate image - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
+        text: ":red-circle: Failed to build and push pay-endtoend candidate image - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
         icon_emoji: ":concourse:"
         username: pay-concourse
     on_success:
       put: slack-notification
       attempts: 10
       params:
-        channel: '#govuk-pay-activity'
+        channel: "#govuk-pay-activity"
         silent: true
-        text: ':green-circle: Built and pushed pay-endtoend candidate image - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
+        text: ":green-circle: Built and pushed pay-endtoend candidate image - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
         icon_emoji: ":concourse:"
         username: pay-concourse
 
   - name: endtoend-e2e
     plan:
       - in_parallel:
-        - get: endtoend-candidate-ecr-registry-test 
+        - get: endtoend-candidate-ecr-registry-test
           passed: [build-and-push-endtoend-candidate]
           trigger: true
           params:
             format: oci
         - get: pay-ci
-      - in_parallel:
         - task: parse-candidate-tag
           file: pay-ci/ci/tasks/parse-candidate-tag.yml
           input_mapping:
             ecr-repo: endtoend-candidate-ecr-registry-test
+        - load_var: release_number_tag
+          file: tags/release-number
```

Comment:
As per the `build-and-push-reverse-proxy-candidate` job below, these `task: parse-candidate` and `load_var` steps will need to be de-indented to come out of the `in_parallel` group above them. The `parse-candidate-tag` needs to wait for the `get: endtoend-candidate-ecr-registry-test` step to finish before it can do anything, so it can't run in parallel with it.